### PR TITLE
2023preBPix campaign

### PR DESCRIPTION
**Pull request by mafrahm**
Note: after merging #49, we need to update the hh2bbvv datasets

### CODE REVIEW

File: cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+#     info=dict(
+#         nominal=DatasetInfo(
+#             keys=[
+#                 "/QCD_PT-300toInf_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+#             ],
+#             n_files=9,
+#             n_events=1582951,
+#         ),
+#     ),
+# )
```
Comment (pkausw) [inline cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py:305]: What's the difference to the dataset above? If we're sure that the one above is correct, I'd propose to delete this part here

```suggestion
```

File: cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+#     info=dict(
+#         nominal=DatasetInfo(
+#             keys=[
+#                 "/QCD_PT-250toInf_bcToE_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+#             ],
+#             n_files=7,
+#             n_events=1663024,
+#         ),
+#     ),
+# )
```
Comment (pkausw) [inline cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py:410]: See comment above

```suggestion
```

**Review by pkausw — CHANGES_REQUESTED** [review]
Nice, thanks @mafrahm ! Just some minor follow-up comments

File: cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py (hunk 1 above)
Comment (mafrahm) [reply cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py:305]: Thank you for the review! I'm not sure where the difference is between the two datasets (except for the name), but the one that I kept had more MC stats compared to this one. But I have no preferences on which one should be kept

**Review by pkausw — APPROVED** [review]
LGTM, thanks!

**Pull request merged**